experimental/desktop_photo_search/lib/main.dart

```diff
@@ -77,6 +78,12 @@ class UnsplashHomePage extends StatelessWidget {
             );
           },
         ),
+        menubar.MenuItem(
+          label: 'Quit',
+          onClicked: () {
+            SystemNavigator.pop();
```

Comment:
Popping out of the top level quits the runner?

Reply:
In my testing under the Windows runner, this doesn't do anything. What am I missing?

Reply:
Interesting. It should quit the app. Or, it does for me... I wonder what's different.

An alternative would be calling exit(), but I'd prefer to avoid it since that would bypass any lifecycle notifications -- when we eventually implement them for desktop platforms. Our docs explicitly warn against calling exit. https://api.flutter.dev/flutter/services/SystemNavigator/pop.html

Reply:
I've just re-tested, this works on macOS (when I re-enable the Quit menu item) but it still fails onn windows. I'm happy to land this as is, but I'm guessing I should raise an issue for the Windows Runner not doing the right thing?